installer/PowerToysSetup/Product.wxs

```diff
@@ -453,6 +455,21 @@
         <Directory Id="ApplicationProgramsFolder" Name="PowerToys (Preview)"/>
       </Directory>
       <Directory Id="DesktopFolder" Name="Desktop" />
+      <?if $(var.PerUser) = "false" ?>
```

Comment:
There's no need for this. PT runner will create data folder if it's not there

Reply:
Updated